explorer/templatetags/explorer_tags.py

```diff
@@ -17,3 +17,11 @@ def export_buttons(query=None):
         'exporters': exporters,
         'query': query,
     }
+
+
+@register.inclusion_tag('explorer/query_favorite_button.html', takes_context=True)
+def query_favorite_button(context, query_id, is_favorite):
```

Comment:
This isn't making use of the context, and I think it'd be useful to allow the tag to add extra classes depending on where it's being used.

```suggestion
def query_favorite_button(query_id, is_favorite, extra_classes):
```